autode/transition_states/truncation.py

```diff
@@ -182,9 +182,9 @@ def add_remaining_atoms(truncated_graph, full_graph, s_molecule):
 
             # Missing atom n from the truncated graph - probably truncated
             # X -> H but was also bonded to another atom also in the truncated
-            # graph
+            # graph.
             x, y, z = s_molecule.atoms[n].coord
-            s_molecule.atoms.append(Atom(atomic_symbol='X', x=x, y=y, z=z))
+            s_molecule.atoms.append(Atom(atomic_symbol='Og', x=x, y=y, z=z))
```

Comment:
Is this going to cause issues in the unlikely situation where someone is using a molecule with Og in it?

Reply:
<img width="673" alt="Screenshot 2020-06-26 at 09 50 54" src="https://user-images.githubusercontent.com/39765193/85839294-a59f2500-b792-11ea-9c24-35df426a8b8b.png">

I think we're okay